ludwig/data/dataset/ray.py

```diff
@@ -130,6 +130,11 @@ def __len__(self):
     def size(self):
         return len(self)
 
+    @property
+    def in_memory_size_bytes(self):
+        memory_usage = self.ds.size_bytes() if not None else 0
```

Comment:
Should be `if self.ds is not None`. As written, this will always resolve to `True`.

Reply:
Hmm, maybe I'm wrong about this. Just tested it and seems to work...

Reply:
So self.ds may be None, but I think even size_bytes() can be none for whatever reason (probably when ds is None)